Example_DynamicFramework/Demo/AppDelegate.m

```diff
@@ -18,8 +18,9 @@
 #import "AppDelegate.h"
 #import "SampleCaseViewController.h"
 #import "AppKeyChoiceViewController.h"
-
 #import <StreetHawkCore/StreetHawkCore.h>
+#import <Analytics/SEGAnalytics.h>
+#import <Analytics/SEGIntegrationsManager.h>
```

Comment:
As the previous comment said, you cannot add segment.io in this way without handling mandatory dependency.

Reply:
this is the demo app, so can we add segment as a dependency ?